keps/NNNN-kep-template/README.md

```diff
@@ -297,13 +297,18 @@ functionality is accessed.
 
 Below are some examples to consider, in addition to the aforementioned [maturity levels][maturity-levels].
 
-#### Alpha -> Beta Graduation
+#### Alpha
+
+- Feature implemented behind a feature flag
+- Initial e2e tests completed and enabled
+
+#### Beta
 
 - Gather feedback from developers and surveys
 - Complete features A, B, C
-- Tests are in Testgrid and linked in KEP
+- Additional tests are in Testgrid and linked in KEP
 
-#### Beta -> GA Graduation
+#### Graduation
```

Comment:
We use the term "graduation" to refer to enhancements moving into beta and alpha too right? Does the use here imply that "graduation" is only for enhancements going stable/GA?

Reply:
you beat me to it. My comment (https://github.com/kubernetes/enhancements/pull/2695#discussion_r628480790) is the same as @JamesLaverack's.